performancetest/src/main/java/org/corfudb/performancetest/CorfuTablePerformanceTest.java

```diff
@@ -0,0 +1,151 @@
+        keyNum = Integer.parseInt(PROPERTIES.getProperty("keyNum", "10"));
+        valueSize = Integer.parseInt(PROPERTIES.getProperty("valueSize", "1024"));
+    }
+
+    private CorfuRuntime initRuntime() {
+        CorfuRuntime.CorfuRuntimeParameters parameters = CorfuRuntime.CorfuRuntimeParameters.builder().build();
+        parameters.setPrometheusMetricsPort(metricsPort);
+        CorfuRuntime corfuRuntime = CorfuRuntime.fromParameters(parameters);
+        corfuRuntime.addLayoutServer(endPoint);
+        corfuRuntime.connect();
+        return corfuRuntime;
+    }
+
+    private CorfuTable<String, String> buildTable(String name) {
+        return runtime.getObjectsView().build()
+                .setTypeToken(new TypeToken<CorfuTable<String, String>>() {})
+                .setStreamName(name)
+                .open();
+    }
+
+    private void putTable(CorfuTable corfuTable, int numRequests) {
+        for (int i = 0; i < numRequests; i++) {
+            String key = keyValueManager.generateKey();
+            String value = keyValueManager.generateValue();
+            corfuTable.put(key, value);
+        }
+    }
+
+    private void getTable(CorfuTable corfuTable, int numRequests) {
+        for (int i = 0; i < numRequests; i++) {
+            String key = keyValueManager.getKey();
+            corfuTable.get(key);
+        }
+    }
+
+    private Thread createThread(CorfuTable table, long start) {
+        return new Thread(() -> {
+            long duration = 0;
+            while ( duration <=  time) {
+                duration = (System.currentTimeMillis() - start) / milliToSecond;
+                try {
+                    putTable(table, random.nextInt(randomBoundary));
+                    getTable(table, random.nextInt(randomBoundary));
+                } catch (Exception e) {
+                    log.error("Operation failed with", e);
+                }
+            }
+        });
+    }
+
+    @Test
+    public void CorfuTable1Thread1Table() {
+        runtime = initRuntime();
+        CorfuTable<String, String>
+                corfuTable = buildTable("table1");
+        long start = System.currentTimeMillis();
+        while (true) {
+            if ((System.currentTimeMillis() - start) / milliToSecond > time) {
+                break;
+            }
+            putTable(corfuTable, random.nextInt(randomBoundary));
+            getTable(corfuTable, random.nextInt(randomBoundary));
+        }
+    }
+
+    @Test
+    public void CorfuTable10Thread1Table() throws InterruptedException {
+        runtime = initRuntime();
+        CorfuTable<String, String>
+                corfuTable = buildTable("table1");
+        Thread[] threads = new Thread[10];
+        long start = System.currentTimeMillis();
+        for (int i = 0; i < 10; i++) {
+            threads[i] = createThread(corfuTable, start);
+            threads[i].start();
+        }
+        for (int i = 0; i < 10; i++) {
+            threads[i].join();
+        }
```

Comment:
m thread n table can be implemented by a helper function with arguments m and n

Reply:
Can you explain more?

Reply:
Create a helper function to avoid duplicate and hard-coded params.
In your 10thread1table, you can directly call helper(10, 1)